stdlib/public/core/BidirectionalCollection.swift

```diff
@@ -45,7 +44,7 @@ extension BidirectionalCollection {
       return _advanceForward(i, by: n)
     }
     var i = i
-    for var offset: IndexDistance = n; offset != 0; offset = offset + 1 {
+    for _ in 0..<(-n) {
```

Comment:
Yup. What you get when trying to watch a two year old while also trying to get a little programming in. (of course that would be one long negative advance to request :))